app/controllers/debt_management_controller.rb

```diff
@@ -7,6 +7,9 @@ def show
   def faq
   end
 
+  def companies
+  end
+
```

Comment:
As far as I remember, you don't need empty controller actions if the only thing they do is rendering a simple view. Correct me if I'm wrong, but I tried commenting out the other actions in this controller and it still did work.